site/_scss/blocks/_compare.scss

```diff
@@ -0,0 +1,52 @@
+// =============================================================================
+// COMPARE OVERVIEW
+//
+// A compare widget is used to show a good/bad variation of something.
+// Typically it's used with a code snippet to show what someone shouldn't do.
+//
+// =============================================================================
+
+.compare {
+  line-height: 1.5em;
+  margin-top: 32px;
+  padding: 0 0 0 16px;
+  position: relative;
+}
+
+.compare--label {
+  font-size: 1.125em;
+  // font-weight: $FONT_WEIGHT_MEDIUM;
```

Comment:
I copied the settings over from web.dev, but I didn't see a system level property for font weight. Seems OK to leave as not set. Thoughts?